code/modules/urist/items/clothes/uristclothes.dm

```diff
@@ -14,6 +14,14 @@ Update 26/07/2014 - All generic clothing goes under obj/item/clothing/under/uris
 	icon = 'icons/urist/items/clothes/clothes.dmi'
 	sprite_sheets = list(SPECIES_RESOMI = 'icons/mob/species/resomi/uristUniform.dmi')
 
+//this is important EXPLICITLY for all the dresses -Vakothu
+/obj/item/clothing/under/urist/dress
```

Comment:
I like what you've done here, but can you leave the yellow dress as a separate subtype? it just makes things a lot cleaner